src/features/game/expansion/components/resources/beehive/Beehive.tsx

```diff
@@ -428,7 +428,7 @@ export const Beehive: React.FC<Props> = ({ id }) => {
             message={[
               {
                 text: t("beehive.pollinationCelebration", {
-                  amount: calculateSwarmBoost(0, gameState),
+                  amount: Number(calculateSwarmBoost(0, gameState).toFixed(2)),
```

Comment:
We have a function that standardises the rounding for all items
```suggestion
                  amount: formatNumber(calculateSwarmBoost(0, gameState)),
```